spec/models/note_spec.rb

```diff
@@ -12,7 +12,7 @@
       it 'with a nil body, return default error message' do
         note = build(:note, body: nil)
         note.save
-        expect(note.errors[:body]).to eq(["Body can't be blank"])
+        expect(note.errors[:body]).to eq(["can't be blank"])
```

Comment:
Might be nice to include a small refactoring to these to match on the contents of the errors rather than the collection itself.

Something like:

`expect(note.errors[:body]).to include('can't be blank')`